apps/console/src/features/applications/components/api-authorization/wizard/authorize-api-resource.tsx

```diff
@@ -110,17 +114,17 @@ export const AuthorizeAPIResource: FunctionComponent<AuthorizeAPIResourcePropsIn
     const [ m2mApplication, setM2MApplication ] = useState<boolean>(false);
     const [ isScopeSelectDropdownReady, setIsScopeSelectDropdownReady ] = useState<boolean>(true);
 
-    const {
-        data: currentAPIResourcesListData,
-        isLoading: iscurrentAPIResourcesListLoading,
-        error: currentAPIResourcesFetchRequestError,
-        mutate: mutatecurrentAPIResourcesList
-    } = useAPIResources(apiCallNextAfterValue);
+    // const {
```

Comment:
IMO, better to remove this entirely. We can bring back with the git history in a later time.

Reply:
Fixed with 9898fbc69e09f8893c765748908a845ed19bcf6b